### PR TITLE
Clarify UG for find and findAny

### LINKED ISSUE #141: [PE-D] Suggestion: syntax of find/findAny command can be confusing to users
Especially the c/ flag part

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: fans2619/ped#6

### PR DESCRIPTION
**Pull request by cookiedan42**
closes #141

### CODE REVIEW

**Review by yucheng11122017 — APPROVED** [review]
LGTM except for small changes!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -231,23 +229,23 @@ Notes:
...
 Notes:
-* `findAny` is case-insensitive by default. e.g. `hans` will match `Hans`.
-  * However, users can opt for case-sensitive search by including the `c/` flag **after** the command word.
-  * `c/` case-sensitive search applies **ONLY** to tags.
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`.
-* Both name and tags are searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`.
-  * This applies to both names and tags. This means `findAny n/an` will not return entries such as `Hans`, but
-    it will return entries such as `Jing An`.
+* `find` is case-insensitive for names. e.g. `n/ hans` will match a contact with name `Hans`.
```
Comment (Jiwei99) [inline docs/UserGuide.md:239]: Should be `findAny` here

File: docs/UserGuide.md [hunk 2]
```diff
@@ -231,23 +229,23 @@ Notes:
...
+* `find` is case-insensitive for tags by default. e.g. `find t/friend` will match a contact with tags `Friend` or `friend`.
```
Comment (Jiwei99) [inline docs/UserGuide.md:240]: Same here, should be `findAny` for both the description and the example

File: docs/UserGuide.md [hunk 3]
```diff
@@ -231,23 +229,23 @@ Notes:
...
+  * Adding the `c/` flag **after** the command word will make tag matching case-sensitive.  
+    e.g. `find c/ t/friend` will match a contact with tag `friend` but not `Friend`.
+* The order of the keywords does not matter. e.g. `n/Hans t/football` will return the same result as `t/friends n/Hans`.
+* Only full words will be matched for names and tags.
+  * `find n/an` will not return entries such as `Hans`, but will return `Jing An`.
```
Comment (Jiwei99) [inline docs/UserGuide.md:245]: Same, `findAny`

**Review by Jiwei99 — APPROVED** [review]

File: docs/UserGuide.md [hunk 4]
```diff
@@ -199,24 +199,22 @@ Notes:
...
-  * This applies to both names and tags. This means `find n/an` will not return entries such as `Hans`, but 
-    it will return entries such as `Jing An`.
-* Only Persons matching all keywords will be returned (i.e. `AND` search).
-  e.g. `n/Hans t/Friend` will return all Persons named `Hans` and are tagged with `Friend`.
+Notes:  
+* `find` is case-insensitive for names. e.g. `n/ hans` will match a contact with name `Hans`.  
+* `find` is case-insensitive for tags by default. e.g. `find t/friend` will match a contact with tags `Friend` or `friend`.  
+  * Adding the `c/` flag **after** the command word will make tag matching case-sensitive.  
+    e.g. `find c/ t/friend` will match a contact with tag `friend` but not `Friend`.
+* The order of the keywords does not matter. e.g. `n/Hans t/football` will return the same result as `t/friends n/Hans`.
```
[the remark this answers is not in the document]
Comment (leezhixuan) [reply docs/UserGuide.md:213]: ^

**Pull request merged**